src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,112 @@
+package com.google.codeu.servlets;
+
+
+import java.io.IOException;
+
+
+import javax.servlet.annotation.WebServlet;
+
+import javax.servlet.http.HttpServlet;
+
+import javax.servlet.http.HttpServletRequest;
+
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+
+import com.google.appengine.api.users.UserServiceFactory;
+
+import com.google.codeu.data.Datastore;
+
+import com.google.codeu.data.User;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+
+/**
+
+ * Responds with a hard-coded message for testing purposes.
+
+ */
+
+@WebServlet("/about")
+
+public class AboutMeServlet extends HttpServlet{
+
+  private Datastore datastore;
+
+  public void init(){
+
+    datastore = new Datastore();
+
+  }
+
+ @Override
+
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+ throws IOException
+ {
+
+   try
+{
+    // statement(s) that might cause exception
+
+
+  response.setContentType("text/html");
```

Comment:
Although it'd change code logic but response.setContentType should come at the end of method. Because setting it in first line is not serving any purpose.